core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -922,6 +923,18 @@ object OracleAttestmentV0TLV extends TLVFactory[OracleAttestmentV0TLV] {
 
     OracleAttestmentV0TLV(eventId, pubKey, sigs, outcomes)
   }
+
+  val dummy: OracleAttestmentV0TLV = {
```

Comment:
This needs to be lazy i'm guessing, I see native image builds are failing, probably because we are eagerly evaluating `ECPrivateKey.freshPrivateKey` which is not allowed when building a native image 

https://github.com/bitcoin-s/bitcoin-s/pull/2656/checks?check_run_id=1952997296#step:5:322